webnn/resources/utils.js

```diff
@@ -825,6 +825,10 @@ const run = async (operationName, context, builder, resources, buildFunc) => {
  * @param {String} deviceType - The execution device type for this test
  */
 const testWebNNOperation = (operationName, buildFunc, deviceType = 'cpu') => {
+  test(() => assert_not_equals(navigator.ml, undefined, "ml property is defined on navigator"));
```

Comment:
I'd like to specially add this **ml** property checking test as a validate test under validation_tests folder, not on here for all invoking this testWebNNOperation function for each operation test.